src/client.ts

```diff
@@ -152,7 +152,10 @@ export class Client {
     if (gameTypesResponse?.status !== 200) throw new Error('Unable to fetch game types static data from data dragon.');
 
     this._seasons = <Season[]>seasonsResponse.data;
-    this._queues = <Queue[]>queuesResponse.data;
+    this._queues = <Queue[]>queuesResponse.data.map((q: Queue) => ({
+      ...q,
+      notes: q.notes ?? undefined // Replace null with undefined for consistency
+    }));
```

Comment:
Idk if I've explained this correctly. This replaces all nulls in queues for undefined so actually typings match result and you don't need to have null here just because it's in the json

Reply:
No need to add comments for stuff like this. But makes sense.